### PR TITLE
Fix bug where models were not listened to when they were set as prope…

### PR DESCRIPTION
**Pull request by bradvogel**
…rties, after initial construction.

### CODE REVIEW

File: lib/connect-backbone-to-react.js [hunk 1]
```diff
@@ -112,6 +115,17 @@ module.exports = function connectBackboneToReact(
       componentWillReceiveProps(nextProps, nextContext) {
         this.setModels(nextProps, nextContext);
         this.createNewProps();
+
+        // Bind event listeners for each model that changed.
+        Object.keys(this.models).forEach(mapKey => {
+          const model = this.models[mapKey];
+          if (this.props.models[mapKey] === this.models[mapKey] ||
```
Comment (hswolff) [inline lib/connect-backbone-to-react.js:122]: We've already done the lookup on `this.models[mapKey]` so let's just use the `model` variable here and in the next line.

Also should we be comparing against `nextProps.models`? As that's the new incoming model?

**Review by hswolff — CHANGES_REQUESTED** [review]
Nice catch! Got a question for you and a change.

File: lib/connect-backbone-to-react.js (hunk 1 above)
Comment (bradvogel) [reply lib/connect-backbone-to-react.js:122]: Sorry for the delay on this! I just pushed a change to DRY up the key lookup.

On `nextProps.models` - it's set as `this.models` by `setModels` by the time this code runs. So we are comparing against `nextProps.models` (or `nextContext.models`, if that's used instead).

Comment (bradvogel) [reply lib/connect-backbone-to-react.js:122]: ping @hswolff

Comment (hswolff) [reply lib/connect-backbone-to-react.js:122]: Aha, got it.

**Comment by hswolff** [comment]
Sorry for the delay, was away on vacation last week!

**Review by hswolff — APPROVED** [review]

**Comment by hswolff** [comment]
Published as `v1.3.0`

**Pull request merged**